pom.xml

```diff
@@ -394,8 +396,8 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-avro-converter</artifactId>
-            <version>5.5.1</version>
-            <scope>test</scope>
+            <version>7.1.1</version>
+            <!--            <scope>test</scope>-->
```

Comment:
remove comments

Reply:
I am not against a version bump here but just want to know the reason. It is entirely fine to just keep it the latest and greatest.

Reply:
With the older version I couldn't use some functions to setup the schema registry client. I don't know if there's better way tho